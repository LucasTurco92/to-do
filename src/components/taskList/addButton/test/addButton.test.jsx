import { render } from '@testing-library/react';
import AddButton from '../addButton';

it('When it invokes the AddButton component, should render it properly', ()=>{

    const { container } = render(<AddButton />);
    
    expect(container).toMatchSnapshot();
});
