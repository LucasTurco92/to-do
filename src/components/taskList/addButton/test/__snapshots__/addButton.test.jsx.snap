// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When it invokes the AddButton component, should render it properly 1`] = `
<div>
  <div
    class="sc-bdvvtL kiJhim"
  >
    <span
      class="sc-gsDKAQ bGJtoD"
    >
      +
    </span>
  </div>
</div>
`;
